api/consumer/swagger.yaml

```diff
@@ -1187,6 +1194,66 @@ components:
           type: string
           pattern: \S*
           description: The reference details for the link.
+    ContentStabilityExtension:
+      type: object
+      properties:
+        url:
+          type: string
+          enum:
+            - "https://fhir.nhs.uk/England/StructureDefinition/Extension-England-ContentStability"
+        valueCodeableConcept:
+          type: object
+          properties:
+            coding:
```

Comment:
Make this a reference instead like ContentStabilityExtensionCoding, so that the model is more explicit instead of CodingItem